source/quickstart/osx.rst

```diff
@@ -6,31 +6,213 @@ Quickstart for Mac OSX
 
 .. default-domain:: minio
 
-This page deploys MinIO on Mac OSX for use with early development
-and evaluation environments.
+.. |OS| replace:: MacOS
 
-For instructions on deploying to production environments,
-see :ref:`deploy-minio-distributed`.
+This page deploys MinIO on |OS| for use with early development and evaluation environments.
+
+For instructions on deploying to production environments, see :ref:`deploy-minio-distributed`.
+
+The procedure described here deploys MinIO in :guilabel:`Standlone Mode`` on a directory on the |OS| harddrive.
+A standalone MinIO deployment consists of a single MinIO server process with a single drive or storage volue (what we call "filesystem mode").
+The MinIO server provides an S3 access layer to the drive or volume.
+
+Objects store to the volume as-is with no :ref:`erasure coding <minio-erasure-coding>`.
 
 Prerequisites
 -------------
 
-ToDo
+- Read, write, and execute permissions for the user's home directory
+- Familiarity with using the Terminal
+
+.. admonition:: Network File System Volumes Break Consistency Guarantees
+   :class: note
+
+   MinIO's strict **read-after-write** and **list-after-write** consistency model requires local disk filesystems (``xfs``, ``ext4``, etc.).
+
+   MinIO cannot provide consistency guarantees if the underlying storage volumes are NFS or a similar network-attached storage volume. 
+
+   For deployments that *require* using network-attached storage, use NFSv4 for best results.
 
 Procedure
 ---------
 
-#. Step 1
+Install the MinIO Server
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Installation with `Homebrew <https://brew.sh>`_ is recommended.
+
+.. tab-set::
+
+   .. tab-item:: Homebrew
+
+      Open a Terminaal and run the following command to install the latest stable MinIO package using `Homebrew <https://brew.sh>`_.
+
+      .. code-block:: shell
+         :class: copyable
+
+         brew install minio/stable/minio
+
+      .. important::
+
+         If you previously installed the MinIO server using ``brew install minio``, then we recommend that you reinstall from ``minio/stable/minio`` instead.
+            
+         .. code-block:: shell
+            :class: copyable
+
+            brew uninstall minio
+            brew install minio/stable/minio
+            
+   .. tab-item:: Binary
+
+      Open a Terminal, then use the following commands to download the standalone MinIO server for MacOS and make it executable.
+            
+      .. code-block:: shell
+         :class: copyable
+
+         wget https://dl.min.io/server/minio/release/darwin-amd64/minio
+         chmod +x minio
+
+Test the :mc:`minio server`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Launch the :mc:`minio server` and use the :ref:`MinIO Console <minio-console>` to work with the MinIO volume.
+
+#. Launch the server
```

Comment:
Having substeps here feels a little out of spec with the previous.

I think we should either do this as three steps:

- Download/Install
- Launch
- Connect

*or* do it as a two-stage setup

"Download and Launch"
"Connect and Explore"